vizro-ai/src/vizro_ai/task_pipeline/pipeline_manager.py

```diff
@@ -0,0 +1,37 @@
+"""Pipeline Manager."""
+from vizro_ai.chains._llm_models import LLM_MODELS
+from vizro_ai.components import (
+    GetChartSelection,
+    GetCustomChart,
+    GetDataFrameCraft,
+    GetVisualCode,
+)
+from vizro_ai.task_pipeline.pipeline import Pipeline
+
+
+class PipelineManager:
```

Comment:
Is this really a pipeline manager, or rather a pipeline collection? Is it best to store this in a class, or would it make more sense to just import pipelines directly as objects?

Reply:
here we need to bridge the llm, llm would only be confirmed when the model is defined when VizroAI is initialized.

about the collection or manager, probably we can make this class to hold more logics in the future, like the for loop we want to plug in can probably also be made here, outside from the main class?

Reply:
Hm could be. I guess as long as it's private, we do not have to worry too much 💪